### DISCUSSION TITLE
Browse Deeplink Issues

### DISCUSSION THREAD

**Discussion by rkato1131** [phantom/docs#77] [General]
I'm using a browse deeplink hooked up to a button in a web application to open a page in Phantom's in-app browser. This has been working fine as long as the user has the Phantom app installed, but for those that don't they get routed to a 404 page on Phantom's website. 

Is this the intended behavior for users who don't have the app installed? I would expect them to be routed to a page to download the application.

**Comment by adamdelphantom** [comment]
Would you mind providing the browse url you are using, please?
